typings/index.d.ts

```diff
@@ -1022,7 +1021,8 @@ export class Interaction extends Base {
   public type: InteractionType;
   public user: User;
   public version: number;
-  public inGuild(): this is this & { guildId: Snowflake; member: GuildMember | APIInteractionGuildMember };
+  public inGuild(bot?: boolean): this is this & { guildId: Snowflake; member: GuildMember | APIInteractionGuildMember };
+  public inGuild(bot: true): this is this & { guildId: Snowflake; member: GuildMember };
```

Comment:
you can also note that `guild` will be present too, because if `bot` is passed it explictly checks if `guild` is not null